juniper_official/Kernel/check-spurious-ppt-wkups.rule

```diff
@@ -0,0 +1,55 @@
+/*
+ * Peer infra module is responsible for replicating FIB states to the line cards
+ * and also to process incoming IPCs from them. Each line card has a connection
+ * with Master Routing Engine Junos Kernel, and a peer proxy kernel thread
+ * represents this remote line card in the Junos Kernel.
+ *
+ * If the peer proxy kernel thread gets woken up spuriously (i.e. without any
+ * pending events to be processed), it can result in performance issues and
+ * should thus be monitored. This rule does exactly that.
+ */
+iceberg {
+    topic kernel.peer-infra {
+        rule check-spurious-ppt-wkups {
+            sensor kernel-peer-infra-sensor {
+                open-config {
+                    sensor-name /junos/kernel/peer-infra;
+                    frequency 1m;
+                }
+            }
+            field spurious-ppt-wkups {
+                sensor kernel-peer-infra-sensor {
+                    path /junos/kernel/peer-infra/spurious-ppt-wkups;
+                }
+                type integer;
+                description "ppt spurious wakeups";
+            }
+            trigger report-spurious-ppt-wkups {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed